kubernetes-itests/pom.xml

```diff
@@ -38,7 +38,7 @@
     </dependency>
     <dependency>
       <groupId>io.fabric8</groupId>
-      <artifactId>crd-generator</artifactId>
+      <artifactId>crd-generator-apt</artifactId>
```

Comment:
Is this dependency need in `kubernetes-itest`? I don't see where or how it's used.

Reply:
I think it's required for CRD Generator tests

https://github.com/fabric8io/kubernetes-client/blob/master/kubernetes-itests/src/test/java/io/fabric8/crd/zookeeper/ZookeeperCustomResourceIT.java

Reply:
Yes, this contains an annotation processor which generates the CRDs from the POJOs. It's not used in the code itself.

Reply:
Ok, I see, but I will need to open an issue about this.